src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java

```diff
@@ -64,8 +62,6 @@ public void selectLanguage(String lang) {
       if (selectedLanguageText.contains(langSelector)) {
         wait.forElementClickable(selectedLanguage);
         selectedLanguage.click();
-
-        Assertion.assertEquals(languageSelectedIndicator.getText(), lang + ".");
```

Comment:
What was wrong with it?

Reply:
1. We only add language-specific prefix to url in supported non-english languages. So if one was to use this method to select "en", this assertion would fail
2. I'm against having assertions in page objects in general (for rationale please see https://martinfowler.com/bliki/PageObject.html, paragraph starting from "There are differences of opinion on whether page objects should include assertions")

Reply:
Should we move the assertion instead of removing it? Ignore if you think it's not valuable or is tested somehow else.